src/cli/CommandsLogic.ts

```diff
@@ -104,11 +104,11 @@ export default class CommandsLogic {
     return response.ready
   }
 
-  async waitForRelay (relayUrl: string): Promise<void> {
-    const timeout = 30
+  async waitForRelay (relayUrl: string, timeout = 60): Promise<void> {
```

Comment:
I don't see where you are using the ability to specify a custom timeout. What was wring with the old code?